datafusion/proto/src/logical_plan.rs

```diff
@@ -385,7 +385,8 @@ impl AsLogicalPlan for LogicalPlanNode {
                 let table_paths = &scan
                     .paths
                     .iter()
-                    .map(|p| ListingTableUrl::parse(p).unwrap())
+                    .filter_map(|p| ListingTableUrl::parse(p).ok())
+                    //.map(|p| ListingTableUrl::parse(p).unwrap())
                     .collect::<Vec<ListingTableUrl>>();
```

Comment:
We don't want to silently ignore errors. How about this approach instead?

```suggestion
                    .map(|p| ListingTableUrl::parse(p))
                    .collect::<Result<Vec<_>, _>>()?;
```